src/components/Containers/PageContainer.tsx

```diff
@@ -7,7 +7,7 @@ export const PageContainer: FC<PropsWithChildren> = ({ children }) => {
       sx={{
         px: { xs: 2, md: 4 },
         mt: 6,
-        mb: { xs: 12, md: 5.5 },
+        pb: { xs: 12, md: 5.5 },
```

Comment:
Not using padding bottom instead of margin will make the bottom navigation appear over the content of the page